source/guide_part-db.rst

```diff
@@ -0,0 +1,161 @@
+   - Installing geertw/ip-anonymizer (v1.1.0): Extracting archive
+   - Installing golonka/bbcodeparser (dev-master a19daa7): Cloning a19daa7456 from cache
+   - Installing ircmaxell/password-compat (v1.0.4): Extracting archive
+   - Installing components/jquery (3.3.1): Extracting archive
+   - Installing jquery-form/form (v4.2.2): Extracting archive
+   - Installing kartik-v/bootstrap-fileinput (v4.5.3): Extracting archive
+   - Installing symfony/polyfill-php72 (v1.11.0): Extracting archive
+   - Installing symfony/polyfill-mbstring (v1.11.0): Extracting archive
+   - Installing symfony/var-dumper (v4.3.2): Extracting archive
+   - Installing maximebf/debugbar (v1.15.0): Extracting archive
+   - Installing nnnick/chartjs (v2.7.3): Extracting archive
+   - Installing psr/http-message (1.0.1): Extracting archive
+   - Installing psr/container (1.0.0): Extracting archive
+   - Installing pimple/pimple (v3.2.3): Extracting archive
+   - Installing nikic/fast-route (v1.3.0): Extracting archive
+   - Installing container-interop/container-interop (1.2.0): Extracting archive
+   - Installing slim/slim (3.12.1): Extracting archive
+   - Installing smarty-gettext/smarty-gettext (1.5.1): Extracting archive
+   - Installing smarty/smarty (v3.1.33): Extracting archive
+   - Installing snapappointments/bootstrap-select (v1.13.10): Extracting archive
+   - Installing squizlabs/php_codesniffer (3.4.2): Extracting archive
+   - Installing tecnickcom/tcpdf (dev-master ca4f9b8): Cloning ca4f9b86c9 from cache
+   0/12 [>---------------------------]   0%
+   8/12 [==================>---------]  66%
+  10/12 [=======================>----]  83%
+  11/12 [=========================>--]  91%
+  12/12 [============================] 100%Generating autoload files
+ [isabell@stardust html]$
+
+You have to create the Part-DB Database before configuring Part-DB like described here :manual_anchor:`additional <database-mysql.html#additional-databases>`:
+::
+
+ [isabell@stardust ~]$ mysql -e "CREATE DATABASE ${USER}_partdb" && echo "Created database ${USER}_partdb"
+ Created database isabell_partdb
+ [isabell@stardust ~]$
+
+
+Now point your browser to your uberspace URL and follow the instructions.
+
+You will need to enter the following information:
+
+  * an initial administrator password: Insert the credentials you want to use for the admin user while installing Part-DB
+  * your MySQL hostname, username and password: the hostname is ``localhost`` and you should know your MySQL :manual_anchor:`credentials <database-mysql.html#login-credentials>` by now. If you don't, start reading again at the top.
+  * your Part-DB database name from the previous step
+
+The installation wizard will prompt you to initialize the database and you will have to change the admin password afterwards.
+
+Updates
+=======
+
+To update Part-DB you copy the data folder to your home directory:
+
+::
+
+ [isabell@stardust ~]$ cp --recursive ~/html/data ~/
```

Comment:
After we copy this directory, the source is deleted anyway. What do you think about changing this to a `mv` instead of `cp`? will be faster, too :)
```suggestion
 [isabell@stardust ~]$ mv ~/html/data ~/
```

Reply:
I would personally keep it as a backup 

And I am thinking about splitting the whole backup part into an own section

Reply:
Oh wait, it's the other direction